aws/resource_aws_ec2_fleet.go

```diff
@@ -0,0 +1,839 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsEc2Fleet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2FleetCreate,
+		Read:   resourceAwsEc2FleetRead,
+		Update: resourceAwsEc2FleetUpdate,
+		Delete: resourceAwsEc2FleetDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"excess_capacity_termination_policy": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  ec2.FleetExcessCapacityTerminationPolicyTermination,
+				ValidateFunc: validation.StringInSlice([]string{
+					ec2.FleetExcessCapacityTerminationPolicyNoTermination,
+					ec2.FleetExcessCapacityTerminationPolicyTermination,
+				}, false),
+			},
+			"launch_template_configs": {
```

Comment:
should be singular

Reply:
nevermind, max 1

Reply:
You were correct that this one should actually be singular, even with max 1, updated. 👍